### ISSUE TITLE
Links must have discernible text

### ISSUE THREAD

**Issue by Khyati007** [khyatit13/Coding-practice#232]
Source: `<a href="https://enterprise.arcgis.com/en/workflow" target="_blank" class="icon-link"><div showdelay="500" class="p-element externalLinks"><calcite-icon icon="question" scale="l" class="svg-icon" aria-hidden="true" calcite-hydrated=""></calcite-icon></div></a>`
Element Location: .icon-link
failureSummary: Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attributeTags: [Ljava.lang.String;@286b39c2
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
